File: src/main/java/gargoyle/ct/config/CTStandardConfigs.java
package gargoyle.ct.config;

import java.util.concurrent.TimeUnit;

public class CTStandardConfigs extends CTConfigs {

    public static final  int  BLOCK_10M        = 10;
    public static final  int  BLOCK_20M        = 20;
    public static final  int  BLOCK_5M         = 5;
    public static final  int  WARN_3M          = 3;
    public static final  int  WHOLE_1H         = 60;
    public static final  int  WHOLE_2H         = 120;
    public static final  int  WHOLE_30M        = 30;
    private static final long serialVersionUID = -2778482225848242583L;

    public CTStandardConfigs() {
        super(new CTConfig(TimeUnit.MINUTES, WHOLE_1H, BLOCK_10M, WARN_3M),
              new CTConfig(TimeUnit.MINUTES, WHOLE_30M, BLOCK_5M, WARN_3M),
              new CTConfig(TimeUnit.MINUTES, WHOLE_2H, BLOCK_20M, WARN_3M));
    }
}

File: src/main/kotlin/gargoyle/ct/config/CTStandardConfigs.kt
package gargoyle.ct.config

import java.io.Serial

object CTStandardConfigs : CTConfigs() {
    const val BLOCK_10M: Long = 10
    const val BLOCK_20M: Long = 20
    const val BLOCK_5M: Long = 5
    const val WARN_3M: Long = 3
    const val WHOLE_1H: Long = 60
    const val WHOLE_2H: Long = 120
    const val WHOLE_30M: Long = 30

    @Serial
    private val serialVersionUID = -2778482225848242583L
}
